packages/my-near-wallet/tsconfig.lib.json

```diff
@@ -6,5 +6,16 @@
     "types": []
   },
   "include": ["**/*.ts"],
-  "exclude": ["**/*.spec.ts"]
+  "exclude": [
```

Comment:
Similar story here to `core` - we could change this to `"exclude": ["jest.config.ts", "**/*.spec.ts"]`